src/current/v25.1/deployment-operations-skills-taxonomy.md

```diff
@@ -0,0 +1,84 @@
+    1. [Shut down a node gracefully]({% link {{ page.version.version }}/node-shutdown.md %})
+    1. Detach the [persistent volume]({% link {{ page.version.version }}/kubernetes-overview.md %}#kubernetes-terminology) (a.k.a. persistent disk) from the removed node's virtual machine (VM) (this step is optional but recommended)
+    1. Delete the removed node's VM
+    1. Start a new VM
+    1. Reattach the persistent disk to the new VM (necessary if you did step #2)
+    1. [Add a node to the cluster]({% link {{ page.version.version }}/cockroach-start.md %}#add-a-node-to-a-cluster) from the new VM
+
+## Troubleshooting
+
+- [SQL response time for specific queries]({% link {{ page.version.version }}/query-behavior-troubleshooting.md %}#query-issues)
+- [SQL throughput degradation across the board]({% link {{ page.version.version }}/query-behavior-troubleshooting.md %}#low-throughput)
+- [Cluster instability: Dead/suspect nodes]({% link {{ page.version.version }}/cluster-setup-troubleshooting.md %}#node-liveness-issues)
+- [Out of memory problems]({% link {{ page.version.version }}/cluster-setup-troubleshooting.md %}#out-of-memory-oom-crash)
+- [Imbalanced cluster load]({% link {{ page.version.version }}/architecture/replication-layer.md %}#load-based-replica-rebalancing)
+- [EOF errors]({% link {{ page.version.version }}/cluster-setup-troubleshooting.md %}#client-connection-issues)
+- [Changefeed is falling behind]({% link {{ page.version.version }}/advanced-changefeed-configuration.md %}#lagging-ranges)
+- [Get a "debug zip" file]({% link {{ page.version.version }}/cockroach-debug-zip.md %})
+- [Get a "tsdump" (timeseries dump) file]({% link {{ page.version.version }}/cockroach-debug-tsdump.md %})
+
+## Disaster recovery
+
+- [Create AWS IAM access key]({% link {{ page.version.version }}/cloud-storage-authentication.md %})
+- [Create S3 bucket for backup data]({% link {{ page.version.version }}/use-cloud-storage.md %}#amazon-s3-storage-classes)
+- [Full cluster backup to S3]({% link {{ page.version.version }}/take-full-and-incremental-backups.md %}#full-backups)
+- [Incremental backup to S3]({% link {{ page.version.version }}/take-full-and-incremental-backups.md %}#incremental-backups)
+- [Cluster restore from AWS S3]({% link {{ page.version.version }}/restore.md %}#restore-a-cluster)
+
+## See also
+
+- [Production Checklist]({% link {{ page.version.version }}/recommended-production-settings.md %})
+- [Deploy CockroachDB Manually]({% link {{ page.version.version }}/manual-deployment.md %})
+- [Deploy a Local Cluster from Binary (Secure)]({% link {{ page.version.version }}/secure-a-cluster.md %})
+- [SQL Performance Best Practices]({% link {{ page.version.version }}/performance-best-practices-overview.md %})
+- [Performance Tuning Recipes]({% link {{ page.version.version }}/performance-recipes.md %})
+- [Troubleshoot Self-Hosted Setup]({% link {{ page.version.version }}/cluster-setup-troubleshooting.md %})
```

Comment:
Should be sentence case

Reply:
these are the names of the page titles being linked to which are using title case per https://github.com/cockroachdb/docs/blob/main/StyleGuide.md#capitalization-rules